src/web/server/locales/en/validation.json

```diff
@@ -16,5 +16,6 @@
   "expectedDeliveryDateTooFarInFutureTitle": "The date you have entered is more than 8 months in the future. You must be at least 10 weeks pregnant to apply for yourself.",
   "expectedDeliveryDateTooFarInFuture": "If you have children under the age of 4, answer ‘no’ to this question to continue with this application on their behalf.",
   "missingPhoneNumber": "Enter a UK mobile number",
-  "invalidPhoneNumber": "Enter in a valid UK mobile number"
+  "invalidPhoneNumber": "Enter in a valid UK mobile number",
+  "invalidEmail": "Enter an email address in the correct format, like name@example.com"
```

Comment:
Shouldn't we have `missingEmail` too as per my comment in the ATs above. Same for Welsh validation.json too.

Reply:
See top comment, we may need a missingEmail message or suffice with just one, such as in enter nino. Discussion needed.

Reply:
This should be driven by Sam Hardie

Reply:
As discussed, sticking with a single invalidEmail message.